content/blog/2021/cff-best-practises.md

```diff
@@ -0,0 +1,71 @@
+
+We are looking forward to seeing beginner, intermediate, and advanced proposals on diverse topics from speakers of different expertise, and experience. To a submit a proposal, you don’t need to be a veteran who has spoken at a dozen conferences in the past. For making our conference truly diverse, we are looking for all kinds of people. That’s what our community is comprised of, and that’s what this conference’s schedule should be truly made out of. As you think about your experiences, skills, and knowledge that you would want to bring to the table. We also want you to focus on the proposal that you will be submitting to us. A well-thought-out proposal could mean a chance for it to be selected and presented at PyCon India 2021. Take your time to go through our team's advice on preparing a kickass proposal that turns heads (virtually):
+
+1. **Know the deadlines, follow the Deadlines.** 
+
+Unlike other iterations of the event, this year we are determined to stick to deadlines that have been enforced and try to provide minimal extensions. Hence, please make note of some important dates as mentioned below:
+
+- CFP opens on **5th March 2021**
+- CFP closes on **7th May 2021**
+- Schedule shall be released on **Last week of July**
+- Talks will be presented online on **18th and 19th September 2020**
+
+Try to follow the deadlines, and don't miss out on submitting the proposal in the first place. Start with [drafting your proposal]((https://in.pycon.org/cfp/2021/proposals/create/)) today.
+
+2. **Just like variable names, titles are important** 
+
+Give your talk a name that accurately describes the focus of that session in regards to potential attendees. Title is going to be the one line elevator pitch for your talk. This is going to be your first impression for both selection committee and audience. Your title will be listed with hundreds of proposals from various speakers when selection committee goes through them, even though selection doesn't solely depend on the title of the talk but remember, first impression is the best impression. Your title will be also listed along with all the talks on the conference day for the audience to go through. They are going to decide which track to attend based on your title. Some of them from last year being, 
+
+1. Under and Dunder - Python secret functions
+2. Taking a peek under the hood - Interpreting black box models
+3. How I built a Ray Tracer in Pure Python
+4. So, You Want to Build an Anti-Virus Engine?
+5. Developing a Single-Sign-On Service using Django
+6. Logging the right way!
+7. Meme Saheb - Using Dank Learning to Generate Original Meme Captions
```

Comment:
Should we also link these proposals, so that people can have some "good" previous proposals to look at along with the titles?

Reply:
Could do it, or folks can find it on the Junction 2020 page, I will drop a link. Resolving.